### PR TITLE
MAISTRA-930: updating the community configuration of grafana to more …

### PR DESCRIPTION
**Pull request by knrc**
…closely match product

### CODE REVIEW

File: grafana/grafana.spec [hunk 1]
```diff
@@ -4,17 +4,22 @@
 %global version 6.2.2
 
 Name:             grafana
-Version:          %{version}
-Release:          3%{?dist}
+Version:          6.2.2
```
Comment (dgn) [inline grafana/grafana.spec:7]: Why hard-code this now?

**Review by jwendell — APPROVED** [review]

File: grafana/grafana.spec [hunk 2]
```diff
@@ -4,17 +4,22 @@
...
+
 # Source0 contains the tagged upstream sources
-Source0:          https://github.com/grafana/grafana/archive/v%{version}.tar.gz
+Source0:          https://github.com/grafana/grafana/archive/v%{version}/grafana-%{version}.tar.gz
 
 # Source1 contains the front-end javascript modules bundled into a webpack
-Source1:          grafana_webpack-%{version}.tar.gz
+Source1:          grafana_webpack-%{version}.%{webpack_hash}.tar.gz
+
+# Source2 is the script to create the above webpack from grafana sources
```
Comment (brian-avery) [inline grafana/grafana.spec:21]: Why was this added? Do we need the commented code?

**Review by brian-avery — APPROVED** [review]

File: grafana/grafana.spec (hunk 2 above)
Comment (knrc) [reply grafana/grafana.spec:21]: To match the product content and make it easier to compare

File: grafana/grafana.spec (hunk 1 above)
Comment (knrc) [reply grafana/grafana.spec:7]: It matches the product configuration and also matches the sed being used in the scripts to update the value should the version change.

We can certainly improve the way it happens in another PR, for both community and product, however the purpose of this PR is to more closely match the product and reduce the deviation.

**Review by dgn — APPROVED** [review]

File: grafana/grafana.spec (hunk 1 above)
Comment (knrc) [reply grafana/grafana.spec:7]: Another reason is the scripts use the value from this line when a version is not explicitly specified so it currently has to be a value rather than reference to the previous global

Comment (dgn) [reply grafana/grafana.spec:7]: Got it, thanks!
